incubator/hnc/test/e2e/issues_test.go

```diff
@@ -209,4 +209,18 @@ var _ = Describe("Issues", func() {
 		// Expected: delete successfully
 		mustRun("kubectl delete subns", nsChild, "-n", nsParent)
 	})
+
+	It("Should create and delete an unpropagatable object - issue 771", func(){
```

Comment:
Oh sorry that I missed this in the first review. It should say "Should **set and unset CannotPropagateObject condition** - issue 771".

Reply:
done